packages/curve-ui-kit/src/widgets/RouteTrack.tsx

```diff
@@ -0,0 +1,20 @@
+import Stack from '@mui/material/Stack'
+import { CircleIcon } from '../shared/icons/CircleIcon'
+
+const Circle = () => <CircleIcon sx={{ width: '0.5rem', height: '0.5rem', fill: 'currentColor' }} />
+
+/**
+ * A vertical track with two circles at the ends and a dotted line in between, used to represent a route.
+ */
+export const RouteTrack = () => (
```

Comment:
It'd be nice to share the whole route component and create storybooks for it. However, out of scope for now.